.circleci/postgresql.conf.sample

```diff
@@ -0,0 +1,15 @@
+# LOGGING
```

Comment:
Should the `.sample` suffix be removed from this file's name? I do see we use it...

Reply:
It likely could be. This came from a tutorial which used the suffix.

Reply:
Not a big deal, was just confused because normally .sample files are intended to be copied and then renamed... but it's fine.